### PR TITLE
Add more docs on running tests

### PR DESCRIPTION
**Pull request by shahzaibj**
Relocating some information from PowerPoints, OneNote as well as add some overview of external technologies used.

### CODE REVIEW

File: docs/Automation/firebase.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+We are going to need login credentials for our Google / Firebase credentials to view the data in there. Those credentials can be obtained via a `KeyVault`.
+
+### (Skip this if you've already done this) How to get access to the KeyVault where Client Secret is stored:
+
+1. Go to myaccess/
+
+2. Click on request access
+
+3. Request permissions from one
```
Comment (iambmelt) [inline docs/Automation/firebase.md:69]: Some strange formatting here.... is information missing? Copy/paste issue?

Comment (shahzaibj) [reply docs/Automation/firebase.md:69]: Copy / Paste :D 

I can do it the proper way, sure.

File: docs/Automation/firebase.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+The service account can be setup by logging in into your Google Cloud Account and creating a service account. After that, you will receive a json file that is essentially the key for this account and allows to authenticate as a service as opposed to a human being i.e. we don't need username / password when using a service account. Our service account is stored in Azure DevOps using ([Secure files for Azure Pipelines and TFS - Azure Pipelines | Microsoft Docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/secure-files?view=azure-devops)).
+
+## Viewing Test Results in Firebase
+
+We are going to need login credentials for our Google / Firebase credentials to view the data in there. Those credentials can be obtained via a `KeyVault`.
+
+### (Skip this if you've already done this) How to get access to the KeyVault where Client Secret is stored:
+
+1. Go to myaccess/
+
```
Comment (iambmelt) [inline docs/Automation/firebase.md:66]: Extra newlines here

File: docs/Automation/firebase.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+4. Click into the `AdalTestInfo` keyvault (if you don't see this KeyVault, make sure you follow the above steps to get access to the KeyVault)
+5. Under settings, click on secrets
+6. Grab the `AndroidFirebaseUsername` - this is username of the Google account
+7. Grab the `AndroidFirebasePassword` - this is password of the Google account
+8. Now you can use these credentials to login into our Google account which would allow us to our Firebase as well as Google cloud portal.
+
+### Firebase Google Account Login Issue
+
+1. When logging in from a new device/location, Google may ask to verify identity.
+
```
Comment (iambmelt) [inline docs/Automation/firebase.md]: Extra newlines here

**Review by iambmelt — APPROVED** [review]

File: docs/Automation/firebase.md (hunk 2 above)
Comment (shahzaibj) [reply docs/Automation/firebase.md]: Addressed here: https://github.com/AzureAD/android-complete/pull/72/commits/671040e8978ceb6c5d05ab3e6c66dd2554cc704c

File: docs/Automation/firebase.md (hunk 3 above)
Comment (shahzaibj) [reply docs/Automation/firebase.md]: Addressed here: https://github.com/AzureAD/android-complete/pull/72/commits/671040e8978ceb6c5d05ab3e6c66dd2554cc704c

**Pull request merged**